lib/engine/round/operating.rb

```diff
@@ -492,7 +492,7 @@ def liquidate(player)
           sell_shares(bundle)
         end
 
-        player.spend(player.cash, @bank)
+        player.spend(player.cash, @bank) if player.cash.positive?
```

Comment:
this is incorrect as well

Reply:
This one is needed for when a player goes bankrupt with zero dollars